src/envo/envo-edit.owl

```diff
@@ -27848,13 +27869,157 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000000> <http://purl.obolibrar
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000006> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000006> "glacial erosion process"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000006> <http://purl.obolibrary.org/obo/ENVO_02500028>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000007> (blue ice)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000007> "Water ice containing little to no air bubbles resulting in a reduction of internal light scattering and, given sufficient volume, a blue appearance.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000007> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000119> <http://purl.obolibrary.org/obo/ENVO_03000007> "https://en.wikipedia.org/wiki/Blue_ice_(glacial)")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000119> <http://purl.obolibrary.org/obo/ENVO_03000007> "https://en.wikipedia.org/wiki/Glacier")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000007> "blue ice"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000007> <http://purl.obolibrary.org/obo/ENVO_01000277>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000008> (compaction process)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000008> "A physical process which forces molecules of a material entity closer together.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000008> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000119> <http://purl.obolibrary.org/obo/ENVO_03000008> "https://en.wikipedia.org/wiki/Compaction")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000008> "compaction process"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000008> <http://purl.obolibrary.org/obo/ENVO_02500000>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000009> (material accumulation process)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000009> "A process during which the mass of one or more materials increases at the same site.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000009> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_03000009> "A material accumulation process ends a material movement process.")
```

Comment:
Can you axiomatise this? I think we have the `ends` relation there. Check the def in RO to make sure it works.